index.js

```diff
@@ -34,9 +38,10 @@ module.exports = function() {
 
       // Unfortunately rework unconditionally splits selectors by comma, so our
       // :matches(...) selector gets munged.
-      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*\)/g, function(substr) {
+      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*?\)/g, function(substr) {
```

Comment:
I'm not saying we should do this, but it could be nice across lots of the rework plugins to standardize paren-matching. I think it's loosely implemented a bunch of different places.

We could pull in something like: https://github.com/dfcreative/parenthesis -- which I'd suppose is the 'node'-y thing to do.